.github/workflows/build_wheels.yml

```diff
@@ -0,0 +1,58 @@
+name: Build
+
+on:
+    pull_request:
+        types: [opened, synchronize]
+    push:
+        tags:
+            - '*'
+
+jobs:
+  build_wheels:
+    name: Build wheels on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        python-version: ['3.6', '3.7', '3.8', '3.9', '3.10']
+        architecture: [x64, x86]
+        exclude:
+            - os: ubuntu-latest
+              architecture: x86
+            - os: macos-latest
+              architecture: x86
+
+    steps:
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
```

Comment:
@kliem I've just discovered that the cibuildwheel step builds wheels already for different python versions (see the matrix at https://cibuildwheel.readthedocs.io/en/stable/options/#build-skip). Thus, it is fine to fix the python version here (say to 3.8).